typescript/sdk/src/models/next_version_to_process.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, PrimaryColumn } from "typeorm";
+import { Base } from "./base";
+
+@Entity("next_version_to_process")
+export class NextVersionToProcess extends Base {
```

Comment:
A note about `NextVersionToProcess` -

We should standardize how we track versions across all the languages. I was planning to update Python to track txn version using the same schema as Rust's `processor_status`. It tracks the latest processed version rather than the next version to process. Let's do the same for TS to standardize.

Reply:
I think latest processed version makes more sense yeah. Can do in a later PR, I'll take notes for all these changes.